README.md

```diff
@@ -33,6 +40,15 @@ Check [our issue tracker](https://github.com/castleproject/Core/issues?utf8=%E2%
 ./build.sh
 ```
 
+## Contributing
+
+There a lot of ways to contribute to the project.
+
+* [![Join the chat at https://gitter.im/castleproject/all](https://badges.gitter.im/castleproject/all.svg)](https://gitter.im/castleproject/all?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+* [Coding Standards](coding-standards.md)
+* [How to submit a fix](how-to-submit-a-fix.md)
+* [Getting Pre-Release Castle Packages](prerelease-packages.md)
```

Comment:
Of course. That was rather stupid.

If I think about it we don't need the Gitter badge here at all. I'll just link to the Home repo from here.